internal/config/config.go

```diff
@@ -218,7 +219,20 @@ func (o *Override) Parse() error {
 		if lastSlash == -1 {
 			return fmt.Errorf("Package override `go_type` specifier %q is not the proper format, expected 'package.type', e.g. 'github.com/segmentio/ksuid.KSUID'", o.GoType)
 		}
-		typename = o.GoType[lastSlash+1:]
+
+		// Upstream typename assignment
+		// typename = o.GoType[lastSlash+1:]
```

Comment:
I left the upstream code as commented out while fiddling around. Those lines should be removed upon an accepted PR to avoid polluting the code.